### ISSUE TITLE
Feature Request: Ability to open up forum channels

### ISSUE THREAD

**Issue by 0xJWLabs** [rigormorrtiss/discordo#455]
Currently, Discordo does not support opening forum channels, limiting users' ability to engage in structured discussions. This feature would enhance user experience by allowing seamless navigation and interaction within forum channels using discordo

**Comment by andy-novikov** [comment]
yes, I've been looking at this project for a while waiting on **forum** functionality.
having that would be lovely.

**Comment by couleurm** [comment]
are you able to enter the channel through a direct channel link or a <#id> link?
